api/src/main/java/org/datacleaner/configuration/RemoteServerState.java

```diff
@@ -22,6 +22,55 @@
 /**
  * States for connection to remote components.
  */
-public enum RemoteServerState {
-    UNKNOWN, OK, ERROR;
+public class RemoteServerState {
+
+    public enum State {
+        UNKNOWN, OK, ERROR, NO_CREDIT;
```

Comment:
Okay. I don't really mind, but NOT_CONNECTED sounds like the user is... well... Not connected :)